build.gradle

```diff
@@ -1,104 +1,129 @@
-plugins {
-	id 'fabric-loom' version '0.9.+'
-	id 'maven-publish'
-	id 'org.springframework.boot' version '2.5.3'
-	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
-	id 'java'
+import com.github.jengelman.gradle.plugins.shadow.transformers.PropertiesFileTransformer
 
+plugins {
+    id 'fabric-loom' version '0.9.+'
```

Comment:
This plugins versions should be in a properties file too

Reply:
Done